### ISSUE TITLE
Enforce rules in `.editorconfig`

### ISSUE THREAD

**Issue by sindresorhus** [SamVerschueren/clinton#77]
You could use https://github.com/editorconfig/editorconfig-core-js to enforce the rules in `.editorconfig`.

**Comment by SamVerschueren** [comment]
I previously tried that but with another tool, I think `eclint`. It didn't work for smart indentation. I will have a look at this one. Thanks!

**Comment by sindresorhus** [comment]
You could only target non-code files where smart indentation is rarely used. I mostly need this for package.json, readme.md, etc.

**Comment by SamVerschueren** [comment]
Good point, will look into it

**Comment by SamVerschueren** [comment]
I found a solution. As suggested by you in #26, I took a look at `lintspaces`. This one works as I want which is pretty neat :)!

I have just one question about the output. The editorconfig linter gives me a line number of an error. But when I render that like XO, it looks like this.

<img width="1101" alt="screen shot 2017-06-16 at 21 18 38" src="https://user-images.githubusercontent.com/1913805/27241564-ae8a1c0e-52d9-11e7-8ec9-820f9cd75980.png">

This looks pretty aweful to me. So I tried with another approach. I strip off the dot at the end of the message at add `at line <NR>`.

<img width="870" alt="screen shot 2017-06-16 at 21 17 54" src="https://user-images.githubusercontent.com/1913805/27241576-bcbac1f2-52d9-11e7-94d0-390c65c42050.png">

Not sure about the last one either. Any suggestions?
